README.md

```diff
@@ -72,3 +80,55 @@ The following make targets are provided:
 The generated Thrift code is committed to the Git repo, so if you change
 `edgecontext.thrift` make sure to run `make thrift fmt` and commit those
 changes as well.
+
+## Go
+
+### Usage
+
+Use [`edgecontext.Factory`] to create an [`ecinterface.Factory`] implementation
+that's expected by [`baseplate.New`]:
+
+```go
+ctx, bp, err := baseplate.New(context.Background(), baseplate.NewArgs{
+  ConfigPath: configPath,
+  ServiceCfg: &cfg, // or nil if you don't have additional config to parse
+  EdgeContextFactory: edgecontext.Factory(edgecontext.Config{
+    Logger: log.ErrorWithSentryWrapper(),
+  }),
+})
+```
+
+When using it, get the [`*EdgeRequestContext`] object out of context:
+
+```go
+if ec, ok := edgecontext.GetEdgeContext(ctx); ok {
+  user := ec.User()
+  loid, ok := user.LoID()
+  // Do something with loid
+}
+```
+
+[`edgecontext.Factory`]: https://pkg.go.dev/github.com/reddit/edgecontext/lib/go/edgecontext#Factory
+[`ecinterface.Factory`]: https://pkg.go.dev/github.com/reddit/baseplate.go/ecinterface#Factory
+[`baseplate.New`]: https://pkg.go.dev/github.com/reddit/baseplate.go#New
+[`*EdgeRequestContext`]: https://pkg.go.dev/github.com/reddit/edgecontext/lib/go/edgecontext#EdgeRequestContext
+
+### Development
+
+We do the same linting checks as Baseplate.go, please follow Baseplate.go's
```

Comment:
I think the fact that there are two "development" sections in the readme is a smell of the problem I'm trying to get at by centralizing this. You and I will be doing per-language changes to the library details, but anyone just making Thrift changes isn't going to want to care about each individual language. What I'm suggesting instead is that we make the dockerfile look something like this:

```dockerfile
# based off the python image just because its install is the most spread out
# and hard to copy
FROM python:3.8-buster

COPY --from=ghcr.io/reddit/thrift-compiler:0.14.1 /usr/local/bin/thrift /usr/local/bin/thrift
COPY --from=docker.io/library/golang:1.16.1-buster /usr/local/go /usr/local/go

WORKDIR /src

COPY go.mod go.sum ./
RUN /usr/local/go/bin/go mod download && \
    /usr/local/go/bin/go install golang.org/x/lint/golint@latest

COPY lib/py/requirements*.txt ./
RUN pip install -r requirements.txt

ENV GOPATH=/root/go
ENV PATH=$GOPATH/bin:/usr/local/go/bin:$PATH

CMD ["/bin/bash"]

```

Then the instructions in the other development section are global and apply to all languages. With ^ dockerfile, I can do the build and run steps as described and then all the make commands work on both languages. No extra configuration is required by the potential contributor.

My goal here is to reduce friction for people wanting to contribute to this stuff as much as humanly possible. Having manual steps from another repo means more headache while having two docker commands to run to set up all languages at once is much easier.